src/app/profile/profile.component.js

```diff
@@ -171,12 +170,11 @@ class ProfileController {
   }
 
   syncPhoneValidators () {
+    const re = /([0-9\s-]{7,})(?:\s*(?:#|x\.?|ext\.?|extension)\s*(\d+))?$/
```

Comment:
Did you write this or copy it from somewhere? A comment with where it came from might be nice. Do we have any common JS files we could import something like this from? You seem to have it in 2 places.